alvr/server/cpp/platform/win32/VideoEncoderNVENC.cpp

```diff
@@ -118,7 +118,7 @@ void VideoEncoderNVENC::Transmit(ID3D11Texture2D *pTexture, uint64_t presentatio
 			fpOut.write(reinterpret_cast<char*>(packet.data()), packet.size());
 		}
 		
-		ParseFrameNals(m_codec, packet.data(), (int)packet.size(), targetTimestampNs, insertIDR);
+		ParseFrameNals(m_codec, packet.data(), (int)packet.size(), targetTimestampNs, picParams.pictureType == NV_ENC_PIC_TYPE_IDR);
```

Comment:
It's here where you would get the encoded bitstream params:

https://github.com/alvr-org/ALVR/blob/0ca1fbc48d977f1df3adaac1c34978fc4184c699/alvr/server/cpp/platform/win32/NvEncoder.cpp#L591-L600

```cpp
bool idr = lockBitstreamData.pictureType == NV_ENC_PIC_TYPE_IDR;
```

Reply:
wouldn't my current change work just fine though?

Reply:
No, because `picParams.pictureType` is never set.

Reply:
I'll do it tomorrow getting a little tired